OpenAI.Utilities/Embedding/EmbeddingTools.cs

```diff
@@ -1,4 +1,6 @@
-﻿using System.Collections;
+﻿#if NET6_0_OR_GREATER
```

Comment:
This utility will not run with .NetStandard 2.0 without more fixes so excluding it from utilities for now.